data-prepper-main/src/main/java/org/opensearch/dataprepper/ContextManager.java

```diff
@@ -26,6 +26,14 @@ public class ContextManager {
 
     private final AnnotationConfigApplicationContext coreApplicationContext;
 
+    private static final String DATA_PREPPER_HOME = System.getProperty("user.dir");
```

Comment:
I'm not sure if this is a reliable way to get data prepper home. An alternative I thought about is to get it from the data prepper main script.

Reply:
I think it would be reasonable for the `data-prepper` script to provide the root Data Prepper directory. It could supply this as a Java property. I wouldn't use `user.dir` however. Perhaps a custom property: `data-prepper.dir`.